### ISSUE TITLE
[OS X] Connecting to network on loading screen.

### ISSUE THREAD

**Issue by Socapex** [ValveSoftware/Dota-2#1687]
This shouldn't happen anymore. http://imgur.com/AKPiKJe

**Comment by Vuccappella** [comment]
This isn't a non windows OS related issue,the dota2 network was down for a moment, nothing unusual. It got fixed in a matter of minutes.

**Comment by Socapex** [comment]
What I meant is, a while back the game was patched so you can still log-in when servers are down, and not see this connecting screen. So issue isn't servers down, but the game stuck on loading instead of in-game client. I can't find the exact changelog, though I've been looking for it.

**Comment by Vuccappella** [comment]
I don't believe that was patched officially, I believe that was only a suggestion by some people, correct me if I'm wrong.
